src/SOS/Strike/strike.cpp

```diff
@@ -13311,7 +13311,11 @@ class ClrStackImplWithICorDebug
         HRESULT Status;
 
         ICorDebugProcess* pCorDebugProcess;
-        IfFailRet(g_pRuntime->GetCorDebugInterface(&pCorDebugProcess));
+        if (FAILED(Status = g_pRuntime->GetCorDebugInterface(&pCorDebugProcess)))
+        {
+            ExtOut("\n\n\n!clrstack -i is unsupported on this target.\nThe ICorDebug interface cannot be constructed.\n");
```

Comment:
Don't use ! in the messages because it isn't used on xplat.  There is a SOSPrefix define that can be used (blank on xplat and ! on Windows). 

Do we really need 3 newlines?

Reply:
I was just looking at it and decided it was too many. It was copied for the line below.  I decided one was enough, but I wanted two at the end...